src/nbl/asset/interchange/CGraphicsPipelineLoaderMTL.cpp

```diff
@@ -78,10 +78,12 @@ void CGraphicsPipelineLoaderMTL::initialize()
     }
 
     // default pipelines
-    auto default_mtl_file = m_assetMgr->getFileSystem()->createMemoryReadFile(DUMMY_MTL_CONTENT, strlen(DUMMY_MTL_CONTENT), "default IrrlichtBAW material");
+    system::ISystem::future_t<core::smart_refctd_ptr<system::CMemoryFile>> future;
+    m_assetMgr->getSystem()->createBufferedFile(future, "default IrrlichtBAW material", system::IFile::ECF_READ);
```

Comment:
change the filename to "Nabla default MTL material" and store it in `constexpr const char*` (we dont want literal constants in the code)